src/components/folder-view.tsx

```diff
@@ -33,54 +40,52 @@ export const FolderView: React.FC<IProps> = ({
 
     return (
         <div>
-            <ul className="mb-3 flex gap-5">
-                {folders.map(folder => (
-                    <button
-                        key={folder.id}
-                        className="flex flex-col items-center rounded-md p-2 hover:bg-black/5"
-                        onClick={() =>
-                            onFolderSelected(decodedDriveID, folder.id)
-                        }
-                    >
-                        <IconFolder className="mb-2" />
-                        <p>{folder.name}</p>
-                    </button>
-                ))}
-            </ul>
-            <table className="w-full">
-                <tbody>
-                    {files?.map(file => (
-                        <tr
+            <ContentSection
+                title={t('folderView.sections.folders.title')}
+                className="mb-4"
+            >
+                {folders.length > 0 ? (
+                    folders.map(folder => (
+                        <FolderItem
+                            key={folder.id}
+                            title={folder.name}
+                            className="w-64"
+                            onClick={() =>
+                                onFolderSelected(
+                                    decodedDriveID,
+                                    encodeID(folder.id)
+                                )
+                            }
+                        />
+                    ))
+                ) : (
+                    <div className="mb-8 text-sm text-gray-400">
+                        {t('folderView.sections.folders.empty')}
+                    </div>
+                )}
+            </ContentSection>
+            <ContentSection title={t('folderView.sections.documents.title')}>
+                {files.length > 0 ? (
+                    files.map(file => (
+                        <FileItem
                             key={file.id}
-                            className="cursor-pointer rounded-md hover:bg-black/5"
+                            title={file.name}
+                            subTitle={`MakerDAO/Ecosystem Actors/Powerhouse/${file.name}`}
+                            className="w-64"
+                            onOptionsClick={() =>
+                                onFileDeleted(decodedDriveID, file.id)
```

Comment:
I'll keep the delete action attached to `onOptionsClick` for now, while we wait for the menu designs